client/src/components/PeopleView/EmployeeModal.js

```diff
@@ -129,12 +133,17 @@ export default class EmployeeModal extends React.Component {
                                     className={`${modalStyles.mood} ${isActive ? modalStyles.activeMood : ''}`}
                                     key={m.color}>
                                     {m.icon}
-                                    <p className={m.color}>{m.text}</p>
+                                    <p className={m.color}>
+                                        {this.props.intl.messages[m.message]}
```

Comment:
Why not use FormattedMessage here? At least get intl out of this.props instead of using it this way.